packages/x-skill/skills-zh/x-chat-provider/SKILL.md

```
@@ -33,9 +33,12 @@ description: 专注于自定义 Chat Provider 的实现，帮助将任意流式
   - [强制规则：禁止自己写 request 方法](#强制规则禁止自己写-request-方法)
 - [⚡ 快速检查清单](#-快速检查清单)
 - [🚨 开发规则](#-开发规则)
-- [参考文件](#参考文件)
+- [🔗 参考资源](#-参考资源)
+  - [📚 核心参考文档](#-核心参考文档)
+  - [🌐 SDK官方文档](#-SDK官方文档)
```

markdownlint-cli2 (0.21.0)

[warning] Link fragments should be valid

(MD051, link-fragments)